src/Typography/Typography.component.tsx

```diff
@@ -72,46 +83,45 @@ export interface TypographyProps extends SpaceProps, React.HTMLAttributes<any> {
     children?: any;
     tag?: TypographyTags;
     weight?: FontWeights;
-    color?: 'inherit' | 'initial' | Color | string;
-    hue?: 'light' | 'base' | 'dark';
+    color?: 'inherit' | 'initial' | string;
     scale?: Scale;
     size?: number;
     lineHeight?: number;
 }
 
 const StyledTypography = (tag: TypographyTags) => styled(tag)<TypographyProps>`
     box-sizing: border-box;
-    // Global Font Properties
-    font-family: ${th('typography.fontFamily')};
-    font-size: ${th('typography.fontSize')};
-    font-weight: ${th('typography.fontWeight')};
-    line-height: ${th('typography.lineHeight')};
+    font-family: base;
+    margin: 0;
 
-    // Use a scale to set size & line-height
-    ${variant({
-        key: 'typography.scale',
-        default: 16,
-        prop: 'scale',
-    })}
-
-    // Apply default styles for element
-    ${variant({
-        key: 'typography.tag',
-        default: 'span',
-        prop: 'tag',
-    })}
+    // Variant styles
+    ${props =>
+        css({
+            fontSize: th('typography.fontSize'),
+            lineHeight: th('typography.lineHeight'),
+            fontWeight: th('typography.fontWeight'),
+            textAlign: 'inherit',
+            color: 'inherit',
+            ...variant({
+                key: 'typography.tag',
+                default: 'none',
+                prop: 'tag',
+            })(props),
+            ...variant({
+                key: 'typography.scale',
+                default: 'none',
+                prop: 'scale',
+            })(props),
+        })}
 
-    // CSS Overrides
-    ${({ align = 'inherit', display, transform = 'none' }) =>
+    // Prop overrides. We don't have any defaults here because then they
+    // would always override the variants above.
+    ${({ color, align, display, transform, lineHeight, size, weight }) =>
         css({
+            color,
             textAlign: align,
```

Comment:
No-op, but I probably should have left `align` as `textAlign`.

Reply:
I was thinking along the same lines. Same goes for `weight` `fontWeight`, `transform` `textTransform`, and `size` `fontSize`.

We could allow both for each, and optionally deprecate.

Reply:
I'll leave that to your discretion. In my mind that makes me wonder (as an engineer) why not just let me alter the CSS via the `style` attribute instead..

Reply:
I'll leave it for now.

Reply:
I was also considering allowing a string for the `size` and `lineHeight` props, to allow for more than just `rem` values.